corehq/toggles.py

```diff
@@ -639,6 +642,7 @@ def _ensure_valid_randomness(randomness):
     TAG_SOLUTIONS_CONDITIONAL,
     help_link='https://confluence.dimagi.com/display/ccinternal/Extension+Cases',
     namespaces=[NAMESPACE_DOMAIN],
+    always_enabled={'icds-cas'}
```

Comment:
There was a ticket about hierarchical fixture on ICDS recently. Possibly related to REACH work.. i have pinged priyanka to check about it. Not a blocker for this PR but will just confirm and update here.

Reply:
Priyanka says they are using hierarchical fixture on ICDS as of now. Just want to flag here if this would effect that.

Reply:
@mkangia I assume you were making this comment about the hierarchical locations toggle (not the extension case sync toggle).

That was one I missed but I've marked it now as 'always enabled': 8845f29235e

Reply:
Ah sorry, i think i got confused by the way the diff was rendered earlier.